package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "smart-home-testing",
+  "version": "1.0.0",
+  "description": "\"Testing library for smart home\"",
+  "main": "src/validate.ts",
+  "directories": {
+    "test": "tests"
+  },
+  "scripts": {
+    "test": "smart-home-testing",
+    "check": "gts check",
+    "clean": "gts clean",
+    "compile": "tsc",
+    "fix": "gts fix",
+    "prepare": "npm run compile",
+    "pretest": "npm run compile",
+    "posttest": "npm run check"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/googleinterns/smart-home-testing.git"
+  },
+  "author": "Google LLC",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/googleinterns/smart-home-testing/issues"
+  },
+<<<<<<< HEAD
```

Comment:
It looks like there are merge conflicts again

Reply:
There still seem to be merge conflicts, as these characters should not exist in a valid JSON file.